alice/MainForm.cs

```diff
@@ -12,6 +12,8 @@ public partial class mainForm : Form
     //-------------------------------------------------------------------------
 
     private bool m_showArchivedProjects = false;
+    private bool m_stayOnTop;
+    System.Windows.Forms.Timer timer = new System.Windows.Forms.Timer();
```

Comment:
Rename var to better explain purpose?
Add a #using so don't need the "System.Windows.Forms"?